packages/strapi-plugin-content-manager/admin/src/components/WysiwygBottomControls/index.js

```diff
@@ -7,37 +9,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { Span, Wrapper } from './components';
-
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
-/* eslint-disable jsx-a11y/no-static-element-interactions */
-
-const WysiwygBottomControls = ({ isPreviewMode, onChange, onClick }) => {
-  const browse = (
-    <FormattedMessage id="components.WysiwygBottomControls.uploadFiles.browse">
-      {message => <Span>{message}</Span>}
-    </FormattedMessage>
-  );
+import { Wrapper } from './components';
 
+const WysiwygBottomControls = ({ onClick }) => {
   return (
     <Wrapper>
-      <div>
-        <label
-          className="dropLabel"
-          onClick={e => {
-            if (isPreviewMode) {
-              e.preventDefault();
-            }
-          }}
-        >
-          <FormattedMessage
-            id="components.WysiwygBottomControls.uploadFiles"
-            values={{ browse }}
-          />
-          <input type="file" onChange={onChange} />
-        </label>
-      </div>
+      <div />
```

Comment:
Is this empty `div` needed?

Reply:
I did not want to change the styling code so yes